RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -441,6 +461,7 @@
                                 detachedQuadStepTracks*
                                 detachedQuadStepClassifier1*detachedQuadStepClassifier2*
                                 detachedQuadStep)
+trackingPhase1.toReplaceWith(DetachedQuadStep, DetachedQuadStep.copyAndExclude([detachedQuadStepHitTriplets]))
```

Comment:
since this is now the default, better eventually migrate to DetachedQuadStep appropriate for CA setup